sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/models/AmqpMessageBody.java

```diff
@@ -53,7 +48,7 @@ public static AmqpMessageBody fromData(byte[] data) {
         Objects.requireNonNull(data, "'data' cannot be null.");
         AmqpMessageBody body = new AmqpMessageBody();
         body.bodyType = AmqpMessageBodyType.DATA;
-        body.data = Collections.singletonList(Arrays.copyOf(data, data.length));
+        body.data = Collections.singletonList(data);
```

Comment:
Why did you stop copying here?

Reply:
We are not providing immutable `AmqpMessageBody` . So I do not copy bytes when user create this object and also in getters. 
Srikanta also commented in his comment above.

https://github.com/Azure/azure-sdk-for-java/pull/17464#discussion_r524866610

Reply:
Based on discussion above https://github.com/Azure/azure-sdk-for-java/pull/17464#discussion_r525535333, I have  removed Arrays.copyOf from  AmqpMessageBody and ServiceBusMessage